trove/instance/service.py

```diff
@@ -519,3 +519,17 @@ def module_remove(self, req, tenant_id, id, module_id):
         if instance_module:
             module_models.InstanceModule.delete(context, instance_module)
         return wsgi.Result(None, 200)
+
+    def guest_eayun_monitor_update(self, req, id, body, tenant_id):
+        """Modify eayun_monitor for an instance."""
+        LOG.debug("req: %s", req)
+        LOG.debug("body: %s", body)
+        if not body or type(body) != dict:
```

Comment:
[unidiomatic-typecheck] Using type() instead of isinstance() for a typecheck. [python/pylint]